atomic_reactor/plugins/pre_change_from_in_df.py

```diff
@@ -92,8 +91,8 @@ def run(self):
                 new_parents.append(df_img)
                 continue
             local_image = self.workflow.dockerfile_images[df_img]
-            # OSBS2 TBD
-            inspection = imageutil.get_inspect_for_image(local_image)
+            # OSBS2 TBD: we don't want to map images to Ids anymore
```

Comment:
but we still want there arch specific image sha

Reply:
Manifest list digest, right? updated

Reply:
right or manifest list digest